### PR TITLE
Delete postinstall

### PR DESCRIPTION
**Pull request by jasonLaster**
This deletes the other post install scripts that we have which will also make this more consistent with playwright and our docs which suggest you run `replayio install`

### CODE REVIEW

**Review by bvaughn — COMMENTED** [review]
Seems like we should also be deleting the "first-run" files in src?

**Comment by jasonLaster** [comment]
good catch. thought i had

File: packages/cypress/package.json [hunk 1]
```diff
@@ -17,7 +17,6 @@
     "*.d.ts"
   ],
   "scripts": {
-    "postinstall": "node ./first-run.js",
```
Comment (Andarist) [inline packages/cypress/package.json:20]: this is a breaking change, so it deserves a major bump in the relevant packages - and we might want to take a look if there are any other breaking changes in those packages that we'd like to batch together with this

Comment (jasonLaster) [reply packages/cypress/package.json:20]: nothing comes to mind. will defer to you.

**Comment by Andarist** [comment]
superseded by https://github.com/replayio/replay-cli/pull/522

**Pull request closed without merging**